redbot/cogs/mod/mod.py

```diff
@@ -104,14 +102,20 @@ def cog_unload(self):
                 await self.settings.guild(discord.Object(id=guild_id)).delete_repeats.set(val)
             await self.settings.version.set("1.0.0")  # set version of last update
         if await self.settings.version() < "1.1.0":
-            prefixes = await self.bot.get_valid_prefixes()
-            prefix = re.sub(rf"<@!?{self.bot.user.id}>", f"@{self.bot.user.name}", prefixes[0])
             msg = _(
                 "Ignored guilds and channels have been moved. "
-                "Please use `{prefix}moveignoredchannels` if "
+                "Please use `[p]moveignoredchannels` if "
                 "you were previously using these functions."
-            ).format(prefix=prefix)
-            self.bot.loop.create_task(self.bot.send_to_owners(msg))
+            )
+            self.bot.loop.create_task(self.bot.send_to_owners_with_prefix_replaced(msg))
+            await self.settings.version.set(__version__)
```

Comment:
This should be changed so that we don't preemptively set the version to latest
```suggestion
            await self.settings.version.set("1.1.0")
```